### DISCUSSION TITLE
New lines in summaries?

### DISCUSSION THREAD

**Discussion by Shakahron** [Difegue/LANraragi#1022] [Ideas]
It's amazing that we can have summeries on galleries now, my library uses them heavily. Is there a limitation that prevents summaries from using new lines? Just trying to set expectations if it's in the plans or not.

**Comment by Difegue** [comment]
New lines are supported server-side, they're just not interpreted in the web client.  
I could quickly fix this by marking the summaries as preformatted text though:  
![image](https://github.com/user-attachments/assets/27659537-a0a5-4d33-b261-5c8b365716cf)
